src/helpers.php

```diff
@@ -5,6 +5,10 @@
 use ArchTech\SEO\SEOManager;
 
 if (! function_exists('seo')) {
+    /**
+     * @template T of string|array
+     * @param T|null $key
```

Comment:
What are these used for?

Reply:
This is called PHP Generics.
https://phpstan.org/blog/generics-in-php-using-phpdocs
But I'm unable to cover all 3 types of calls on this method...

Reply:
I know they're generics, I'm just wondering how they're used in specific here.

Reply:
I've started this PR with this: using generics in the `seo` function.
It has 3 functionalities. I've failed to write proper PHPDoc :(

Reply:
We'll be merging one more PR and then I'll let you know the final feedback on this 👍🏻

Reply:
Okay, so should we keep the generics here? If it's not a full implementation then they should be removed.

Reply:
Done.